locust/static/locust.js

```diff
@@ -40,15 +40,19 @@ $(".close_link").click(function(event) {
     $(this).parent().parent().hide();
 });
 
-$("ul.tabs").tabs("div.panes > div").on("onClick", function(event) {
-    if (event.target == $(".chart-tab-link")[0]) {
-        // trigger resizing of charts
-        rpsChart.resize();
-        responseTimeChart.resize();
-        usersChart.resize();
-    }
+$("ul.tabs").tabs("div.panes > div").on("onClick", function (event) {
+    // trigger resizing of charts
+    resizeCharts();
 });
 
+var charts = []
+function resizeCharts() {
```

Comment:
Will this resize the charts when the window is resized?

Reply:
Yes, the same as it did before—it's triggered only if you click on a different tab then click back on the tab with the charts.